build.ps1

```diff
@@ -0,0 +1,121 @@
+<#
```

Comment:
Cake released a dotnet global tool with the version 0.30.0 which would allow to remove these bootstrap code and the `build.sh`.

https://cakebuild.net/blog/2018/08/cake-v0.30.0-released

Reply:
Not sure if we want to do that already in the first PR. Right now, contributors who want to review this only need to install .NET Core SDK and Docker to get started.
I suppose we introduce the Cake dotnet global tool in a subsequent PR.

Reply:
I created #3 for this. So, we can have a separate PR for this which probably increases the awareness of contributors for this change.